src/scss/grommet-core/_objects.layer.scss

```diff
@@ -183,6 +183,16 @@
       }
     }
   }
+
+  .#{$grommet-namespace}tiles {
+    overflow: auto;
```

Comment:
Any reason not to have this always be a part of tiles, even when outside of a layer?